src/Messenger/KafkaTransportFactory.php

```diff
@@ -70,7 +70,8 @@ public function createTransport(string $dsn, array $options, SerializerInterface
             $options['topic']['name'],
             $options['flushTimeout'] ?? 10000,
             $options['receiveTimeout'] ?? 10000,
-            $options['commitAsync'] ?? false
+            $options['commitAsync'] ?? false,
+            $options['ackFailedMessages'] ?? true
```

Comment:
Can we actually set the default to false here? The `reject()` call is intended for messages that have failed to process after all.

The Exceptions that cause Messenger to call `reject()` should be caused by the applications message handlers.

Could the same effect be achieved by the application implementing a "CatchAllExceptionsMiddleware"?